### PR TITLE
init changes

### PR DESCRIPTION
**Pull request by DanCassiano**

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -29,13 +30,16 @@ import { getFixedTokens } from './src/services/contentful'
 import NavigationService from './src/utils/hocs/NavigationServices'
 // import './ReactotronConfig'
 
-if (!__DEV__) {
+if (!__DEV__) { // eslint-disable-line
   Sentry.config('https://8ffba48a3f30473883ba930c49ab233d@sentry.io/1236809', {
```
Comment (dbuarque) [inline App.js]: Need remove this to variable

File: src/scenes/Balance/AccountsCarousel/index.js [hunk 2]
```diff
@@ -62,38 +62,52 @@ export class AccountsCarousel extends React.Component {
     const { accounts } = this.props.context
     const createdItemPosition = accounts.length - 1
     // We set the state to load before the item is focused
-    this._onSnapToItem(createdItemPosition)
+    // this._onSnapToItem(createdItemPosition)
```
Comment (pedrobertao) [inline src/scenes/Balance/AccountsCarousel/index.js]: remove commented  code

File: src/scenes/Balance/AccountsCarousel/index.js [hunk 3]
```diff
@@ -62,38 +62,52 @@ export class AccountsCarousel extends React.Component {
...
+      { text: tl.t('remove'), onPress: () => this._handleHideAccount(address, balance) }
     ])
   }
 
-  _handleHideAccount = async address => {
+  _handleHideAccount = async (address, balance) => {
     try {
       const { pin, hideAccount } = this.props.context
-      const nextAccountIndex = this.currentIndex() - 1
+      // const nextAccountIndex = this.currentIndex() - 1
```
Comment (pedrobertao) [inline src/scenes/Balance/AccountsCarousel/index.js]: remove commented code

File: App.js [hunk 4]
```diff
@@ -281,18 +284,23 @@ class App extends Component {
...
+  }
 
   _hideAccount = address => {
     const newAccounts = this.state.accounts.filter(acc => acc.address !== address)
     this.setState({ accounts: newAccounts })
   }
 
+  _getCurrentAccount = () => {
+    const { publicKey, accounts } = this.state
+    return accounts.find(account => account.address === publicKey)
```
Comment (pedrobertao) [inline App.js]: we have a lot of problems about this specific find to return undefined. It would be greate if we could handle it here

**Pull request merged**